.github/workflows/labels-issues-prs.yml

```
@@ -0,0 +1,83 @@
+    types: [opened, reopened, transferred]
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      # Apply path‑based labels for PRs via .github/labeler.yml
+      - name: Run PR labeler
+        if: github.event_name == 'pull_request'
+        uses: actions/labeler@v4
```

actionlint (1.7.7)

the runner of "actions/labeler@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)